src/utils/constants.ts

```diff
@@ -45,6 +45,8 @@ export const denominationNames: { [key in Denomination]: string } = {
 export const BasePorts = {
   bitcoind: {
     rest: 18443,
+    zmqBlock: 28334,
+    zmqTxn: 28335,
```

Comment:
I suggest setting this to `29335`. In a local testing environment, I don't think it's a big deal for the ports to be way off from the default values. You won't need to do the workaround in `createBitcoindNetworkNode()`. With the ports spaced 1000 apart, you practically avoid any chance of conflict. 

I know this is not intuitive, but it was the best I came up with at the time. I left a comment on `BasePorts` stating "_These should be sufficiently spaced apart to allow a dozen or so numbers higher and not cause conflicts_", but I guess that could be more explanatory.

Reply:
Changed to this solution instead.

Oh right, I missed that comment. I should've read more carefully.